src/native-common/Image.tsx

```diff
@@ -101,20 +101,23 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         }
 
         const additionalProps = this._getAdditionalProps();
+        const extendedProps: RN.ExtendedImageProps = {
+            source: imageSource,
+            shouldRasterizeIOS: this.props.shouldRasterizeIOS,
+            tooltip: this.props.title
+        };
 
         return (
             <RN.Image
-                ref={ this._onMount }
-                style={ this.getStyles() }
-                source={ imageSource }
-                resizeMode={ resizeMode }
+                ref={ this._onMount as any }
+                style={ this.getStyles() as RN.StyleProp<RN.ImageStyle> }
+                resizeMode={ resizeMode as any }
                 resizeMethod={ this.props.resizeMethod }
                 accessibilityLabel={ this.props.accessibilityLabel }
-                onLoad={ this.props.onLoad ? this._onLoad : undefined }
+                onLoad={ this.props.onLoad ? this._onLoad as any : undefined }
```

Comment:
Based on [`ImageProps` ](https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageProps.js#L32) `onLoad` does not accept any arguments, hm... 

```
onLoad?: ?() => void,
```

Reply:
Yeah, that's a bug in the type definition file. If you look at the RN.Android and RN.IOS sources, you'll see that it returns the dimensions.